flow-typed/types/text-config.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import {TextStyle} from '../../src/track/text-style';
+
+declare type PKTextConfigObject = {
+  enableCEA708Captions: boolean,
+  useNativeTextTrack: boolean,
+  textTrackDisplaySetting: Object,
+  textStyle: TextStyle,
+  forceCenterCaptions: boolean,
```

Comment:
I think this will apply to subtitles as well so better to make this more generic and call it `forceCenter`?

Reply:
sounds good